ospd_openvas/daemon.py

```diff
@@ -1360,6 +1439,8 @@ def exec_scan(self, scan_id: str):
                 time.sleep(0.05)
             got_results = False
 
+        # Sleep a second to be sure to get all notus results
+        time.sleep(1)
```

Comment:
The problem is, that we wait 0.25 seconds after receiving a notus result, before publishing all results we got in this time. I tested it without this sleep and sometimes I did not get all notus results. I don't think this will be a problem at all, as we simply wait another second at the end of a scan, which normally take some minutes to several hours.